CONTRIBUTING.md

```diff
@@ -185,78 +110,32 @@ This is to ensure that:
 * your PR is easy to browse and review
 * git log is easier to digest
 
-#### Example
+Also, in general your contribution will fall into one of two categories:
 
-* Bad:
+1. A fix
 
-```
-Refactor X.
+In this case, structure your commit like below:
 
-Changelog: None
 ```
+fix: <description of the fix>
 
-```
-Refactor Y to use X.
-...but also, fix some more of X!
+<More detailed explanation of the commit>
 
-Changelog: None
+Changelog: <None|Title|Commit|All>
+Ticket: <None|Ticket Nr>
 ```
 
-```
-Add tests for X and Y.
-
-Changelog: None
-```
-
-```
-Fix even more of X and Y to make the tests pass, d'oh!
-
-Changelog: None
-```
 
-These commits reflect an ordinary workflow of incremental
-changes and fixes, but they are unwieldy for review because of
-(un)related changes being scattered all over.
 
-Use rebase with edits and squashes to rework this into something more cohesive:
+* A new feature
 
-* Good:
 ```
-Refactor and test X.
-
-Changelog: None
-```
-
-```
-Refactor Y to use X, test Y.
-
-Changelog: None
-```
-
+feat: <description of the new feature>
 
-...or even:
-```
-Refactor X.
-
-Changelog: None
-```
-
-```
-Add tests for X.
-
-Changelog: None
-```
-
-```
-Refactor Y to use X.
-
-Changelog: None
-```
-
-```
-Add tests for Y.
+<More detailed explanation of the commit>
 
-Changelog: None
+Changelog: <None|Title|Commit|All>
+Ticket: <None|Ticket Nr>
 ```
```

Comment:
Should we top it off by linking to https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md somewhere in this file? So far it's only in the pull request template.

Reply:
Good point!